### DISCUSSION TITLE
noreply@mail.app.supabase.io

### DISCUSSION THREAD

**Discussion by steynkalala** [supabase/supabase#2514] [Questions] [answered]
is it possible to change the "noreply@mail.app.supabase.io" email ?

**Comment by dshukertjr** [comment, marked as answer]
Hi @SudoStain!

You can achieve this by setting up custom SMTP in your Supabase console:
<img width="1360" alt="Screen Shot 2021-07-24 at 7 34 51" src="https://user-images.githubusercontent.com/18113850/126847647-32273414-2fb3-4795-8625-c0a16d5972e3.png">

**Reply by steynkalala** [reply]
I changed it to my SendGrid credentials but when the confirm link is clicked it redirects to a  SendGrid verification url for example:  url1111.domain.com.  do I need a paid account to properly use this feature or do I have to us Gotrue? 
<img width="1929" alt="Screen Shot 2021-07-24 at 11 47 04 AM" src="https://user-images.githubusercontent.com/10123821/126873878-f7266ad1-3c5c-4f03-927c-273a77b426dc.png">

<img width="1468" alt="Screen Shot 2021-07-24 at 11 52 49 AM" src="https://user-images.githubusercontent.com/10123821/126874012-b151b5b9-973c-4927-b03a-28ad8601f797.png">

**Reply by dshukertjr** [reply]
Hey @SudoStain !

I tried this with Sendgrid right now, and was able to make it work with free version of Sendgrid and Supabase. 

Your setup seems correct to me. For SMTP password, you are using your API key, correct? The fact that the confirmation link takes you to Sendgrid verification screen might be a bit suspicious. Could you confirm that the `SMTP Admin Eamil` you have entered in your Supabase dashboard is a verified email address on Sendgrid?

<img width="1360" alt="Screen Shot 2021-07-25 at 10 28 54" src="https://user-images.githubusercontent.com/18113850/126884850-cf977333-afff-4ef6-a583-3936fa119a3b.png">

**Comment by steynkalala** [comment]
Hi @dshukertjr 
Yes I'm using the api key and the STMP Admin email in Supabase is the verified email. So I created a new SendGrid account and tried it with a gmail extension(somename@gmail.com) and the verify link works, Of course I'd to use my own domain extension. To me it seems like the Sendgrid branding link process is effecting the apps verification link redirect because its redirects to a link that SendGrid requires to be added to the domain DNS records for link branding...the solution was to delete the link branding, create a new api key(the other api key is somehow associated with the branded url even if you delete the url record from your DNS) and use the new api key on Supabase, after that you can rebrand the link and everything works. Thanks for helping me solve this!
